deploy/kubernetes/base/csidriver.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
   name: fsx.csi.aws.com
 spec:
   attachRequired: false
+  fsGroupPolicy: ReadWriteOnceWithFSType
```

Comment:
Based on the documentation: https://kubernetes-csi.github.io/docs/support-fsgroup.html it seems like not specifying the fsGroupPolicy policy will default to ReadWriteOnceWithFSType. Is this change needed & could this have any impact on any usecases for csi driver?